### PR TITLE
Add Message.Attachment#getContentType

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [x] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

See https://github.com/discord/discord-api-docs/pull/2762

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java [hunk 1]
```diff
@@ -1283,8 +1283,9 @@ public MessageReaction createMessageReaction(MessageChannel chan, long id, DataO
         final String url = jsonObject.getString("url");
         final String proxyUrl = jsonObject.getString("proxy_url");
         final String filename = jsonObject.getString("filename");
+        final String contentType = jsonObject.getString("content-type", null);
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java:1286]:
```suggestion
        final String contentType = jsonObject.getString("content_type", null);
```

File: src/main/java/net/dv8tion/jda/api/entities/Message.java [hunk 2]
```diff
@@ -2282,6 +2284,18 @@ public String getFileExtension()
             return index == 0 || index == fileName.length() ? null : fileName.substring(index);
         }
 
+        /**
+         * The Content-Type of this file.
+         * <br>This is the  <a href="https://en.wikipedia.org/wiki/Media_type" target="_blank">Media type</a> of the file that would be used in an HTTP request or similar.
```
Comment (caneleex) [inline src/main/java/net/dv8tion/jda/api/entities/Message.java:2289]:
```suggestion
         * <br>This is the <a href="https://en.wikipedia.org/wiki/Media_type" target="_blank">Media type</a> of the file that would be used in an HTTP request or similar.
```

**Comment by Andre601** [comment]
I assume this could simplify the isImage check (or whatever it wss called).
At least for when the content type is there.

File: src/main/java/net/dv8tion/jda/api/entities/Message.java (hunk 2 above)
Comment (DV8FromTheWorld) [reply src/main/java/net/dv8tion/jda/api/entities/Message.java:2289]: @MinnDevelopment This doesn't appear like it was actually incorporated. Why was it marked `resolved`?

Comment (MinnDevelopment) [reply src/main/java/net/dv8tion/jda/api/entities/Message.java:2289]: Because it changes nothing of value.

**Pull request merged**